woocommerce/woocommerce
# Deploy spotlight product tour

### All Submissions:






### Changes proposed in this Pull Request:



Closes 92-gh-woocommerce/team-ghidorah.

- Deploy the treatment and replace the control code.
- Clean-up remaining experiment code.

### How to test the changes in this Pull Request:

1. Use a fresh site
2. Navigate to `Tools > WCA Test Helper > Experiments`
3. Set `woocommerce_products_task_layout_card_v3` and `woocommerce_products_task_layout_stacked_v3` to control
4. Go to Products > All products
5. Click on "Create Product" button
6. Observer that spotlight product tour is shown.
7. Go to WooCommerce > Home > Add products
8. Click on "Start with a template" and choose a template
9. Confirm that spotlight product tour is shown.
10. Repeat 5~7 steps with different templates
11. Repeat 5 step and choose "Add manually" option.
12. Confirm that spotlight product tour is shown.

### Other information:




### FOR PR REVIEWER ONLY:
